glue/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+      // This package uses the Function type, and it will take effort to fix.
+      "ban-types": false
```

Comment:
for new packages, this file should only `"extends": "dtslint/dt.json"`, you can disable some rules in source code like `// tslint:disable-next-line ban-types`